### PR TITLE
[Feature] Bounding boxes change default domain, and class_label->class_id

### PR DESCRIPTION
**Pull request by nbardy**
1) This changes the domain of bounding box position values to be from 0 to 1.
2) Moving the pixel coordinate system to a `domain: actual` property.
3) Add a nice new test that renders four boxes in each corner in the four different style to test the are all the same in the renderer.
4) Change `class_label` to `class_id`

Results of the test:
image

### CODE REVIEW

File: wandb/data_types.py [hunk 1]
```diff
@@ -1008,22 +1008,39 @@ def validate(self , boxes):
...
                    valid = True
                 elif has_num(box["position"], "minX") and \
                      has_num(box["position"], "maxX") and \
                      has_num(box["position"], "minY") and \
                      has_num(box["position"], "maxY"):
+                   vals = [box["position"]["minX"],
+                           box["position"]["maxX"],
+                           box["position"]["minY"],
+                           box["position"]["maxY"]]
+                   if not all([(0 <= v <= 1) for v in vals]) and box.get("domain") != "actual":
```
Comment (shawnlewis) [inline wandb/data_types.py]: I don't think we want to reject these. Models will happily make predictions outside the image.

Comment (nbardy) [reply wandb/data_types.py]: Good catch. Removed the checks
